pppd/crypto.c

```diff
@@ -176,12 +231,21 @@ int PPP_crypto_deinit()
         g_crypto_ctx.provider = NULL;
     }
 #endif
+
+#ifdef PPP_WITH_OPENSSL
+    ERR_free_strings();
+#endif
     return 1;
 }
 
 #ifdef UNIT_TEST
 #include <stdio.h>
 
+int debug;
```

Comment:
Including utils.c to use the error() function requires access to a few variables not included in the compilation for utest_crypto check program. Ideally, we should have cleaned up all the uses of debug, but unfortunatey they are still used in downstream plugins and it would break the abi